src/Microsoft.DotNet.Interactive.Formatting/TabularDataFieldList.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright(c).NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Interactive.Formatting
+{
+    public class TabularDataFieldList : List<TabularDataSchemaField>
```

Comment:
These types should not be public.

Reply:
i made them public to be used in the data frame extensions, so we can reuse such schema objects

Reply:
I'd have to see the use case but I suspect we should not support this here.

Reply:
making those internal until we need them